cookbooks/aws-parallelcluster-entrypoints/recipes/update.rb

```diff
@@ -11,6 +11,7 @@
 # or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
 # OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
 # limitations under the License.
+include_recipe "aws-parallelcluster-shared::setup_envars"
```

Comment:
We should capture the recipe inclusion in `update_spec.rb`.
Same for init and finalize.